.github/pull_request_template.md

```diff
@@ -1,15 +1,15 @@
-#### Pull request checklist
-
-- [ ] Addresses an existing issue: Fixes #0000
-- [ ] Added relevant unit test for your changes. (`npm run test`)
-- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
-- [ ] Ran precheckin (`npm run precheckin`)
-- [ ] Added screenshots/GIFs for UI changes.
-
 #### Description of changes
 
 (give an overview. Add technical description describing your changes)
 
 #### Notes for reviewers
```

Comment:
I thought we agreed the "Notes for reviewers" section as well; did I misremember?

Reply:
Updated!